tools/zetasql_helper/zetasql_helper/fixer/fix_column_not_grouped.cc

```diff
@@ -0,0 +1,160 @@
+//
+// Copyright 2020 BigQuery Utils
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "fix_column_not_grouped.h"
+
+#include "absl/strings/str_cat.h"
+#include "zetasql/parser/parser.h"
+#include "zetasql_helper/util/util.h"
+
+
+namespace bigquery::utils::zetasql_helper {
+
+
+absl::Status
+FixColumnNotGrouped(absl::string_view query, absl::string_view missing_column, int line_number, int column_number,
```

Comment:
Can you use absl::StatusOr<string> if it is released?  so you don't have pass output as string*?

Reply:
absl::StatusOr is not released, but I use the one defined in ZetaSQL - zetasql_base:StatusOr<T>